var/spack/repos/builtin.mock/packages/aml/package.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+class Aml(AutotoolsPackage):
```

Comment:
Should we call this `libtool-dependent`, since it has nothing to do with aml anymore?